2020 channel updates

cbituin opened the pull request:
This PR will resolve issue #41 . 

This PR separates channels into three major categories: General, Development/Language-Specific, and Regional/Location-based.

General channels - No channels were removed; **some** channels that were above 50 members were added. 

Development/Language-Specific - No channels were removed; **some** channels that were above 50 members were added.

Regional/Location-based - I've added all the channels I recognized as location-based channels. 

This list is not meant to be all-inclusive, because _wow_ there are a lot of channels!

![image](https://user-images.githubusercontent.com/38743840/76153496-71441380-6092-11ea-959f-f5a34a2a5f46.png)

![image](https://user-images.githubusercontent.com/38743840/76153503-828d2000-6092-11ea-90bf-09c7f28f325c.png)

![image](https://user-images.githubusercontent.com/38743840/76153507-88830100-6092-11ea-872e-11ffbbfc56a1.png)

kylemh commented:
Looks as though you have merge conflicts to resolve @cbituin

calincionca35 commented:
Are the three categories going to be listed separately, and if so, how will they be accessed (maybe via separate tabs)? Or will the three categories be merged into one list?

cbituin commented:
> Are the three categories going to be listed separately, and if so, how will they be accessed (maybe via separate tabs)? Or will the three categories be merged into one list?

@calincionca35 I took the original 'master list' and split the list into three different tables. The list might be a little less user friendly since .md tables are also not able to be sorted or filtered. The guide has been in a markdown file along with the other docs, so I don't see this changing anytime soon.

Your question did get me thinking about usability, though. So I added a small, linked TOC that can be expanded if needed. 

As far as tabs, I am not sure how this would be done in .md. Seems like a great idea, but also could get a bit cramped if someone wanted to keep categorizing.
